examples/OpenTracing.Examples/CommonRequestHandler/RequestHandler.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using OpenTracing.Tag;
+
+namespace OpenTracing.Examples.CommonRequestHandler
+{
+    // One instance per Client. Executed concurrently for all requests of one client. 'beforeRequest'
+    // and 'afterResponse' are executed in different threads for one 'send'
+    public class RequestHandler
+    {
+        internal const String OperationName = "send";
+
+        private readonly ITracer _tracer;
+
+        private readonly ISpanContext _parentContext;
+
+        public RequestHandler(ITracer tracer) : this(tracer, null)
+        {
+        }
+
+        public RequestHandler(ITracer tracer, ISpanContext parentContext)
+        {
+            _tracer = tracer;
+            _parentContext = parentContext;
+        }
+
+        public void BeforeRequest(Object request, Context context)
```

Comment:
"object" instead of "Object" (last comment regarding this - pls fix everywhere - same for "string" vs "String")